src/test/java/hudson/security/PAMSecurityRealmTest.java

```diff
@@ -25,8 +25,8 @@ public void testLoadUsers() {
             // expected
         }
 
-        String name = LIBC.getpwuid(LIBC.geteuid()).pw_name;
-
+        POSIX api = PosixAPI.jnr();
+        String name = api.getpwuid(api.geteuid()).getLoginName();
         System.out.println(Arrays.asList(sc.userDetails.loadUserByUsername(name).getAuthorities()));
```

Comment:
Does this seem to work when run interactively on Linux? I do not think the test suite really checks anything on CI.

Reply:
This works locally and shows all the groups my user is in.
And yes, there are no adequate tests for this plugin.
At least I verified in the Jenkins UI with my local user that
a) with a valid password I can log in
b) with an invalid password I cannot log in